### PR TITLE
Update grammar in Docker deprecation blog posts

### PR DESCRIPTION
**Pull request by bl-ue**
Fix grammar

### CODE REVIEW

File: content/en/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ as a container runtime after v1.20.
 
 **You do not need to panic. It’s not as dramatic as it sounds.**
 
-tl;dr Docker as an underlying runtime is being deprecated in favor of runtimes
```
Comment (sftim) [inline content/en/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:16]: The original is idiomatic, uh, web forum slang.

File: content/en/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md [hunk 2]
```diff
@@ -95,10 +95,10 @@ So, this change is coming. It’s going to cause issues for some, but it isn’t
 catastrophic, and generally it’s a good thing. Depending on how you interact
 with Kubernetes, this could mean nothing to you, or it could mean a bit of work.
 In the long run, it’s going to make things easier. If this is still confusing
-for you, that’s okay -- there’s a lot going on here, Kubernetes has a lot of
+for you, that’s okay&mdash;there’s a lot going on here; Kubernetes has a lot of
 moving parts, and nobody is an expert in 100% of it. We encourage any and all
 questions regardless of experience level or complexity! Our goal is to make sure
-everyone is educated as much as possible on the upcoming changes. `<3` We hope
```
Comment (sftim) [inline content/en/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:101]: > `<3` 

The authors intended to communicate a :heart: here - I'd like to leave that in somehow.

**Review by sftim — COMMENTED** [review]
Overall, nice fixes but I'm loth to see the `<3` removed as cutting it out changes the article's sentiment.

@bl-ue can you reword? Maybe check with the article authors.

File: content/en/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md (hunk 1 above)
Comment (bl-ue) [reply content/en/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:16]: @sftime I'll make it lowercase again.  I'm used to seeing it uppercase in some places.

File: content/en/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md (hunk 2 above)
Comment (bl-ue) [reply content/en/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:101]: @sftim how about putting it at the end? e.g.:

> We hope this has answered most of your questions and soothed some anxieties! `<3`

Comment (bl-ue) [reply content/en/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:101]: Also we could just use the emoji itself:
> We hope this has answered most of your questions and soothed some anxieties! ❤️

**Comment by bl-ue** [comment]
> I'm loth to see the <3 removed as cutting it out changes the article's sentiment.

Based on where it was in the sentence I almost thought it was mistake.  I removed it partially to confirm that it wasn't. ;)

I agree with you that it doesn't read well where it is currently.  See my suggestions above (https://github.com/kubernetes/website/pull/25462#discussion_r537708434 & https://github.com/kubernetes/website/pull/25462#discussion_r537787736.)

File: content/en/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md (hunk 2 above)
Comment (bl-ue) [reply content/en/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:101]: @sftim WDYT?

Comment (mrbobbytables) [reply content/en/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:101]: I'd be okay with the heart 👍

Comment (bl-ue) [reply content/en/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md:101]: @mrbobbytables Done. PTAL.

**Review by onlydole — APPROVED** [review]
Thank you, @bl-ue! This looks good to me! 🤘🏼😄

**Comment by bl-ue** [comment]
Hi @sftime, looks like you've been around recently, could you please lgtm this PR?

**Pull request merged**